recipes/cuda-python/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
+{% set patch_version = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - setuptools
+    - cython
+    - pycparser
+    - autopxd2
+    - pyclibrary
+    - python
+    - pip
+  run:
+    - python
+    - cudatoolkit >={{ major_version ~ ".0.0" }}
+  run_constrained:
+    - __cuda >={{ major_version ~ ".0.0" }}
```

Comment:
As `cudatoolkit` has [this constraint]( https://github.com/conda-forge/cudatoolkit-feedstock/blob/28a3595be00c218da2dc6cff2fc70ef4151f11ec/recipe/meta.yaml#L300-L301 ) as well and we have now added it as a dependency, think we can drop this

Reply:
Done